ssbh_data/src/meshex_data.rs

```diff
@@ -11,8 +11,9 @@ use geometry_tools::bounding::{
 use itertools::Itertools;
 #[cfg(feature = "serde")]
 use serde::{Deserialize, Serialize};
+use ssbh_lib::formats::mesh::BoundingSphere;
```

Comment:
This should be `pub use` so that people using ssbh_data can create a `BoundingSphere`. This is only used in one module, so exporting it from `ssbh_data::meshex_data` instead of the crate root like `Vector4` is fine.